processor/routingprocessor/metadata.yaml

```diff
@@ -11,3 +11,27 @@ status:
 
 tests:
   config:
+
+telemetry:
+  metrics:
+    routing_processor_non_routed_spans:
```

Comment:
I wonder if these metrics should be configured as `optional: true` since they're only ever needed if the different signal is configured in the pipeline

Reply:
I just tried this, but I think I hit a bug in mdatagen: if all metrics are optional, `err` is unused in `NewTelemetryBuilder` causing a compilation error. I'll keep this as is for now, and try to address that bug in another PR.

Reply:
@jpkrohling have you already opened an issue for this?